source/administration/changelog.md

```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.8
+
+Release Date 16/02/2019
+
+- Mattermost v5.8.0 contains low to medium level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Highlights
+
+#### Added support for LDAP Group Sync
+ - Lets admins set default team and channel membership based on LDAP groups. See more details [in the documentation](https://docs.mattermost.com/deployment/ldap-group-sync.html).
+
+#### Added multi-factor authentication support to Team Edition
+ - See more details on [this Forum post](https://forum.mattermost.org/t/multi-factor-authentication-mfa-in-team-edition/6287).
+ 
+#### Enhanced image performance
+ - Improved performance for images by adding support for image proxy servers, which are now integrated into the server and switched on by default.
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved sorting of emoji in the emoji autocomplete and emoji picker search results.
+ - Added support for emoji picker for mobile web view.
+
+#### Notifications
+ - Added a channel notification setting to disable at-channel mentions.
+ 
+#### Administration
+ - Added the ability to search users by role in **System Console** > **Users**.
+ - Added a CLI command to modify an outgoing webhook.
+ - Added a CLI command to restore a team.
+
+#### Performance
+ - Made posting idempotent.
+
+#### Slash Commands
+ - Added support for sending a message to a different channel than where the slash command was issued from.
+ - Improved error handling of slash commands.
+ - Added an option to send a message when an invalid slash command is entered on the right-hand side.
```

Comment:
Added an option to send a message beginning with a "/" from the right-hand side.